src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -83,9 +161,9 @@ private RazorSemanticTokensLegend()
             TokenTypes = new Container<string>(_tokenTypes),
         };
 
-        public Container<string> TokenTypes { get; private set; }
+        public static Container<string> TokenTypes { get; private set; }
```

Comment:
Does this need to be public? Having a public static that's a mutable list makes me squirm 😄. Ditto with the modifiers

Reply:
Not only does it not need to be public, it doesn't need to exist lol.